### PR TITLE
Spring cleanup

### PR DESCRIPTION
**Pull request by bilgili**

### CODE REVIEW

File: livre/lib/types.h [hunk 1]
```diff
@@ -39,22 +39,22 @@ struct EFPrefetchAlgorithmParameters;
...
+typedef std::shared_ptr< const TextureUploadProcessor > ConstTextureUploadProcessorPtr;
+typedef std::shared_ptr< const DashProcessor > ConstDashProcessorPtr;
+typedef std::shared_ptr< const TextureDataObject > ConstTextureDataObjectPtr;
+typedef std::shared_ptr< const TextureObject > ConstTextureObjectPtr;
+typedef std::shared_ptr< TextureDataObject > TextureDataObjectPtr;
+typedef std::shared_ptr< TextureObject > TextureObjectPtr;
 
 /** Map definitions */
-typedef boost::unordered_map< uint32_t, DataUploadProcessorPtr > DataUploadProcessorPtrMap;
+typedef std::hash_map< uint32_t, DataUploadProcessorPtr > DataUploadProcessorPtrMap;
```
Comment (eile) [inline livre/lib/types.h]: std::hash_map!? Shouldn't that be std::unordered_map?

**Comment by eile** [comment]
+1

Spring is good.

File: livre/lib/types.h (hunk 1 above)
Comment (bilgili) [inline livre/lib/types.h]: ops, the type is not even used anymore

**Comment by bilgili** [comment]
please retest this

**Comment by bilgili** [comment]
retest this please

**Pull request merged**